app/views/help/email_filtering.md

```diff
@@ -4,7 +4,7 @@ Yes. You’ll need to set up a filter in Gmail.
 
 ## First, select the C2 request messages to filter.
 
-1. Enter the following into the search bar at the top of your inbox: `from:communicart.sender@gsa.gov`
+1. Enter the following into the search bar at the top of your inbox: `from:c2.sender@gsa.gov`
```

Comment:
There is no email for this. I can set it up. We will need to also login/create users on servers when this gets pushed.